recipe/meta.yaml

```diff
@@ -34,22 +34,24 @@ requirements:
     - libtool        # [unix]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [osx]
+    - libgfortran  # [linux]
   run:
     - zlib 1.2*
+    - libgfortran  # [not win]
 
 test:
   requires:
-    - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
-    - python 3.5.*  # [win and py35]
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Ah, but I'm suggesting we make it a test requirement. Travis CI is complaining we don't have `gfortran`. So this should fix it.